t/50reverse-proxy-timings.t

```diff
@@ -11,6 +11,7 @@ plan skip_all => 'curl not found'
 
 plan skip_all => 'mruby support is off'
     unless server_features()->{mruby};
+diag(`curl --version`) if $ENV{TEST_DEBUG};
```

Comment:
Do we want to print version of curl and other test programs *always* and somewhere else (e.g., at the beginning of the CI or at the beginning of `make check`)?